README.md

```diff
@@ -0,0 +1,23 @@
+Welcome to Maria Durrani's Portfolio! 🚀
+
+Hey there! 👋 Welcome to my portfolio, where I showcase my journey as a budding frontend developer! 🌟
+
+About Me ℹ️  
+I'm passionate about creating beautiful and intuitive user experiences through web development. Whether it's crafting elegant designs or diving into the intricacies of code, I love every aspect of frontend development! 💻
+
+What You'll Find Here 📂   
+Projects: Explore my latest creations, from simple landing pages to interactive web applications.   
+Skills: Check out the technologies I'm proficient in, including HTML, CSS, JavaScript, and more!   
+About Me: Learn more about my background, interests, and aspirations in the world of frontend development.
+
+Let's Connect! 🤝
+
+I'm always eager to collaborate and learn from fellow developers! Feel free to reach out to me via:
```

Comment:
Heyyy nice job. I think it makes sense to also mention the list of technologies that we are working with in this project. Also some patterns that we use, maybe some architecture as well :)